packages/attribute-registry-process/src/services/attributeRegistryService.ts

```diff
@@ -70,6 +74,36 @@ export const attributeRegistryService = {
       })
     );
   },
+  async createCertifiedAttribute(
+    apiCertifiedAttributeSeed: ApiCertifiedAttributeSeed,
+    authData: AuthData
+  ): Promise<string> {
+    const certifier = await getCertifier(authData.organizationId);
+
+    return repository.createEvent(
+      createCertifiedAttributeLogic({
+        attribute: await readModelService.getAttributeByCodeAndName(
```

Comment:
Here there are two awaited promises that could be parallelized with a `Promise.all`, what do you think?

Reply:
Nice catch, I applied your suggestion!